charts/flink-job/README.md

```diff
@@ -51,6 +51,11 @@ See metrics reporter in the flink properties for more details.
 | podMonitor.sampleLimit | int | `2000` | (`int`) Per-scrape limit on number of scraped samples that will be accepted. |
 | podMonitor.scrapeInterval | string | `"15s"` | (`string`) The frequency in which to scrape metrics. |
 | pvc | object | `{"storage":"1Gi","storageClassName":"efs"}` | Configuration of the PersistentVolume for storing savepoints. |
+| pvcRocksdb | object | `{"enabled":false,"rocksdbDir":"/rocksdb","storage":"10Gi","storageClassName":"gp3-encrypted"}` | Configuration of rocksdb PVC |
+| pvcRocksdb.enabled | bool | `false` | (`boolean`) whether to enable RocksDB PVC |
+| pvcRocksdb.rocksdbDir | string | `"/rocksdb"` | (`string`) The mount path of the rocksdb volume |
+| pvcRocksdb.storage | string | `"10Gi"` | (`string`) The size of the storage |
+| pvcRocksdb.storageClassName | string | `"gp3-encrypted"` | (`string`) The class name of the storage. The list of available storage classes is from `k get storageclasses` |
```

Comment:
1. Remove the default storage-class name here ... its internal to us, and is not a relevant default for anyone else.
2. You don't need a storageClassName here actually at all - if you don't pick one, the cluster uses its default. You _can_ put the option in, but if so, the general pattern is to not set it to anything explicit, and instead make sure the helm chart user sets it to something that is relevant to them.